src/commands/appdistribution-group-create.ts

```diff
@@ -0,0 +1,17 @@
+import { Command } from "../command";
+import * as utils from "../utils";
+import { requireAuth } from "../requireAuth";
+import { AppDistributionClient } from "../appdistribution/client";
+import { getProjectName } from "../appdistribution/options-parser-util";
+
+export const command = new Command("appdistribution:group:create <displayName> [alias]")
+  .description("create group in project")
+  .before(requireAuth)
+  .action(async (displayName: string, alias?: string, options?: any) => {
+    const projectName = await getProjectName(options);
+    const appDistroClient = new AppDistributionClient();
+    utils.logBullet(`Creating group in project`);
+    const group = await appDistroClient.createGroup(projectName, displayName, alias);
+    alias = group.name.split("/").pop();
+    utils.logSuccess(`Group '${group.displayName}' (${alias}) created successfully`);
```

Comment:
Thoughts on adding `alias = ` or something like that to the string to make it clear what's in the parentheses? So that if a dev then goes to presumably add the alias when uploading, it's clear which string to use

Reply:
Done.